src/services/balancer/subgraph/entities/pools/helpers.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { parseUnits } from '@ethersproject/units';
+
+import decoratedPool from './__mocks__/decoratedPool';
+
+import {
+  ExcludedAddresses,
+  removeAddressesFromTotalLiquidity
+} from './helpers';
+
+describe('helpers', () => {
+  describe('removeAddressesFromTotalLiquidity tests', () => {
+    const mockedDecoratedPool = {
+      ...decoratedPool,
+      totalShares: 1000,
+      totalLiquidity: 2000
+    };
+
+    it('Should correctly remove excluded addresses', () => {
+      const excludedAddresses: ExcludedAddresses = {
+        [mockedDecoratedPool.address]: {
+          '0x359f4fe841f246a095a82cb26f5819e10a91fe0d': parseUnits('250'),
+          '0x001c249c09090d79dc350a286247479f08c7aad7': parseUnits('250')
+        }
+      };
```

Comment:
The idea here is to remove `500` shares of the `1000` available ones which should result in a reduction of 50% of the `totalLiquidity` (AKA the `miningTotalLiquidity`)